src/components/RichLayout/index.js

```diff
@@ -4,13 +4,16 @@ import Layout from 'components/Layout'
 import Header from './Header'
 import Footer from './Footer'
 
+import style from './style.css'
+
 class RichLayout extends React.Component {
   static propTypes = {
     className: PropTypes.string
   }
 
   render () {
-    return (<div>
+    return (
+      <div className={style.richLayout}>
         <Header />
 
         <Layout className={this.props.className}>
```

Comment:
We have too much margin between the header and the breadcrumbs. (Because remaining space is equally distrubeted between `<Header />` `<Layout />` and `<Footer />`. To solve this, wrap `<Header />` and `<Layout />` in a seperate div, so we have
``` javascript
<div className={style.richLayout}>
 <div>
  <Header />
  <Layout className={this.props.className}>{this.props.children}</Layout>
 </div>
 <Footer />
</div>
```

Reply:
done